Extensions/Video/JsExtension.js

```diff
@@ -0,0 +1,775 @@
+      .getCodeExtraInformation()
+      .setFunctionName("isPaused");
+
+    object
+      .addExpression(
+        "IsPaused",
+        _("Video is played"),
+        _("Return if video is played"),
+        _("Time"),
+        "res/physics16.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("isPaused");
+
+    object
+      .addExpression(
+        "IsPlayed",
+        _("Video is played"),
+        _("Return if video is played"),
+        _("Time"),
+        "res/physics16.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("isPlayed");
+
+    object
+      .addCondition(
+        "Loop",
+        _("Is looped"),
+        _("Is looped"),
+        _("_PARAM0_  is looped"),
+        "Control time",
+        "JsPlatform/Extensions/videoicon24.png",
+        "JsPlatform/Extensions/videoicon16.png"
+      )
+      .addParameter("object", _("Choose a video object"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("isLooped");
+
+    object
+      .addExpression(
+        "IsPaused",
+        _("Video is paused"),
+        _("Return if video is paused"),
+        _("Time"),
+        "res/physics16.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("isPaused");
+
+    object
+      .addExpression(
+        "IsLooped",
+        _("Video is looped"),
+        _("Return if video is looped"),
+        _("Time"),
+        "res/physics16.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("isLooped");
+
+    object
+      .addCondition(
+        "Volume",
+        _("Volume"),
+        _("Test the volume of an video object"),
+        _("Volume of _PARAM0_ is _PARAM1_ _PARAM2_"),
+        "Volume",
+        "JsPlatform/Extensions/videoicon24.png",
+        "JsPlatform/Extensions/videoicon16.png"
+      )
+      .addParameter("object", _("Choose a video object"), "", false)
+      .addParameter("relationalOperator", _("Sign of the test"))
+      .addParameter("expression", _("Value 0-100"))
+      .getCodeExtraInformation()
+      .setFunctionName("getVolume")
+      .setManipulatedType("number");
+
+    object
+      .addCondition(
+        "Mute",
+        _("Is muted"),
+        _("Test if an video is muted"),
+        _("_PARAM0_  is muted"),
+        "Volume",
+        "JsPlatform/Extensions/videoicon24.png",
+        "JsPlatform/Extensions/videoicon16.png"
+      )
+      .addParameter("object", _("Choose a video object"), "", false)
+      .getCodeExtraInformation()
+      .setFunctionName("isMuted");
+
+    object
+      .addExpression(
+        "GetCurrentTime",
+        _("Get current time"),
+        _("Return the current time of an video object (in seconds)"),
+        _("Time"),
+        "res/physics16.png"
```

Comment:
All icons must be replaced by `JsPlatform/Extensions/videoicon16.png `

Reply:
Why use two icons when only the first one is used?
     
ExampleJsExtention.js  L34
```
"res/conditions/camera24.png",
"res/conditions/camera.png"
```
(Can i set just one path for my video ext?)